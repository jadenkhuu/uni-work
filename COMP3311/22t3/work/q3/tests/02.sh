psql racing -c "select * from q3('Oaks') order by race;"
