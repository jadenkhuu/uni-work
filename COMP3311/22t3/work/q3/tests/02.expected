      race       |        horses        
-----------------+----------------------
 Australian Oaks | F5,G2,G3,M3,M3,M3,M4
 Queensland Oaks | F4,G2,G5,M3,M4,M4,M4
 VRC Oaks        | F4,G3,M3,M3,M3,M3
(3 rows)
